lib/trento_web/views/v2/cluster_view.ex

```diff
@@ -6,11 +6,26 @@ defmodule TrentoWeb.V2.ClusterView do
   end
 
   def render("cluster.json", %{cluster: cluster}) do
+    cluster_details = render("cluster_details.json", %{details: cluster.details})
+
     cluster
     |> Map.from_struct()
+    |> Map.put(:details, cluster_details)
+    |> Map.delete(:deregistered_at)
     |> Map.delete(:__meta__)
   end
 
+  def render("cluster_details.json", %{details: %{"sap_systems" => sap_systems} = details}) do
+    sap_systems =
+      Enum.map(sap_systems, &Map.take(&1, [:distributed, :filesystem_resource_based, :nodes]))
```

Comment:
What is this for? to remove the `sid` field?
The sid cannot be removed, it is used in the frontend.
Yet again, most probably the schema is what it is wrong